nvflare/tool/job/config/configer.py

```diff
@@ -226,56 +228,45 @@ def parse_cli_config(cli_configs: List[str], app_names: List[str], job_folder) -
     return app_cli_config_dict
 
 
-def build_config_file_indices(job_folder: str) -> Dict[str, Dict[str, Tuple]]:
-    excluded = ["info"]
-    included = ["config_fed_client", "config_fed_server", "meta"]
+def build_config_file_indices(job_folder: str, app_names: List[str]) -> Dict[str, Dict[str, Tuple]]:
+    config_included = ["config_fed_client", "config_fed_server"]
```

Comment:
For these string constants, should we put them in a share place?

I think we have somewhere else also using these strings as well. (Also "app", "config")?